dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -245,15 +245,15 @@ class VersionedFile(NamedTuple):
 
 
 AIRFLOW_PIP_VERSION = "25.0.1"
-AIRFLOW_UV_VERSION = "0.6.13"
+AIRFLOW_UV_VERSION = "0.6.16"
 AIRFLOW_USE_UV = False
 # TODO(potiuk): automate upgrades of these versions (likely via requirements.txt file)
 GITPYTHON_VERSION = "3.1.44"
 RICH_VERSION = "13.9.4"
 PRE_COMMIT_VERSION = "4.2.0"
 HATCH_VERSION = "1.14.0"
 PYYAML_VERSION = "6.0.2"
-UV_VERSION = "0.6.13"
+UV_VERSION = "0.6.16"
```

Comment:
Lets fix this one at least -- this module already imports other things from `airflow_breeze.global_constants`, so lets import `UV_VERSION` from there.
```suggestion
```

Reply:
Yeah just fixed a push

Reply:
pushed a fix*